style.css

```diff
@@ -1,62 +1,226 @@
-body {
-  background: url('aust copy.jpg');
-  background-size: cover;
-  /* position: fixed; */
+/* JS Navbar Scrolling CSS */
+.navbar.scrolled {
+	background: #a1cbdf;
+	transition: background 400ms;
 }
-
-.splash {
-  font-family: 'Noto Sans', sans-serif;
-  padding-top: 60px;
+#city {
+	background: url('building.png');
+	background-size: cover;
+	min-height: 100vh;
+	position: relative;
+}
+#BRAND {
+	font-size: 1.2rem;
+	padding-top: 0;
+	padding-bottom: 0;
+}
+#NL {
+	font-size: 1rem;
+	padding-top: 2px;
+	padding-bottom: 2px;
+}
+/* --------------------------------------- SECTION 1 --------------------------------------------------------------- */
+.nameDev {
+	margin-top: 0;
+	margin-bottom: 0;
+}
+.links {
+	color: #a0ffff;
+}
+.links:hover {
+	color: #a0ffff;
+}
+.socialList {
+	padding-top: 1rem;
+	text-align: center;
+	color: #a0ffff;
+	font-size: 1.5rem;
+}
+#github {
```

Comment:
These all have the same style.
Turn these into one class name, like `.social-media-links` instead of creating 3 ids.
Use classes as much as possible, id only when necessary.

Reply:
I messed up.  I am working on the testBranch still so you were commenting on less recent stuff.  Sorry man.